content/en/docs/concepts/workloads/controllers/deployment.md

```diff
@@ -17,10 +17,10 @@ weight: 30
 A _Deployment_ controller provides declarative updates for [Pods](/docs/concepts/workloads/pods/pod/) and
 [ReplicaSets](/docs/concepts/workloads/controllers/replicaset/).
 
-You describe a _desired state_ in a Deployment object, and the Deployment controller changes the actual state to the desired state at a controlled rate. You can define Deployments to create new ReplicaSets, or to remove existing Deployments and adopt all their resources with new Deployments.
+You describe a _desired state_ in a Deployment, and the Deployment controller changes the actual state to the desired state at a controlled rate. You can define Deployments to create new ReplicaSets, or to remove existing Deployments and adopt all their resources with new Deployments.
```

Comment:
I think in this first sentence, `... Deployment object,` helps indicate that Deployment is a resource, as well as a Controller,  and that you can define this resource in k8s configuration.

Reply:
The style guide mentions not to use  'Deployment object'.. instead to say the Deployment.  That's why I removed it. Is the update ok in that sense? 
[1] https://kubernetes.io/docs/contribute/style/style-guide/#use-camel-case-for-api-objects